packages/datetime/src/dateRangePicker.tsx

```diff
@@ -155,25 +162,30 @@ export class DateRangePicker
 
     public render() {
         const modifiers = combineModifiers(this.modifiers, this.props.modifiers);
-        const { className, locale, localeUtils, maxDate, minDate } = this.props;
+        const { className, isSequential, locale, localeUtils, maxDate, minDate } = this.props;
         const isShowingOneMonth = DateUtils.areSameMonth(this.props.minDate, this.props.maxDate);
         const { leftView, rightView } = this.state;
         const { disabledDays, selectedDays } = this.states;
 
-        if (isShowingOneMonth) {
-            // use the left DayPicker when we only show one
+        if (isSequential || isShowingOneMonth) {
```

Comment:
we can add another prop with `numberOfMonths` yup (though we can discuss on a new issue thread) and the logic should just work out of the box - the css will need some tweaking to support that